test-series/0/Validation_040.ipynb

```diff
@@ -0,0 +1,648 @@
+{
```

Comment:
<p>To me, after about 400 realizations, it looks like the residuals are roughly flat, rather than converging (maybe slightly). Is there an argument for how fast we should <em>expect</em> it to converge? Also, the plot legend seems to be discrepant with the caption (sigma=1mK vs. sigma = 0.5).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera-validation/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>